Rubberduck.VBEEditor/SafeComWrappers/VB6/VBProjects.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Rubberduck.VBEditor.SafeComWrappers.VB6
 {
-    public class VBProjects : SafeComWrapper<VB.VBProjects>, IVBProjects
+    public class VBProjects : SafeEventedComWrapper<VB.VBProjects, VB._dispVBProjectsEvents>, IVBProjects
     {
         private static readonly Guid VBProjectsEventsGuid = new Guid("0002E190-0000-0000-C000-000000000046");
```

Comment:
same as above

Reply:
The problem is that we use the interface (`IVBComponents` and `IVBProjects`), which are shared by both VB6 and VBA, so if I don't implement VB6 using same interface, I get compilation errors because the consuming code can't call `AttachEvents` / `DetachEvents` since VB6's implementation won't have `SafeEventedComWrapper` interface. I agree it's lousy but short of actually deleting the whole VB6 implementation, I'm not sure what I can do about that. I will at least add a TODO to warn it's all wrong.

Reply:
Ah right, understood.  Yeah, a TODO will suffice.